storage/state-view/src/account_with_state_view.rs

```diff
@@ -5,21 +5,21 @@ use aptos_types::{
     account_address::AccountAddress, account_view::AccountView, state_store::state_key::StateKey,
 };
 
-pub struct AccountWithStateView<'a> {
+pub struct AccountWithStateView<'a, S> {
```

Comment:
Why do you need to change this?

Reply:
didn't have to, some leftover, got rid of this and the corresponding test change, thanks! 2 less files